library/core/functions.render.php

```diff
@@ -858,7 +858,7 @@ function filtersDropDown($baseUrl, array $filters = [], $extraClasses = '', $def
             // Add the default link to the top of the list.
             array_unshift($links, [
                 'active' => $active === null,
-                'name' => $default,
+                'name' => t($default),
```

Comment:
I would update the default to `null` (instead of "All") and translate it at the beginning of the function. That way, we're only translating text generated in this function.

```php
function filtersDropDown($baseUrl, array $filters = [], $extraClasses = '', $default = null, $defaultUrl = null, $label = 'View') {
    if ($default === null) {
        $default = t('All');
    }
```